src/components/IdeaInfoOverlay/IdeaInfoOverlay.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import IdeaInfoOverlay from './';
+
+describe('IdeaInfoOverlay', () => {
+	it('renders without crashing', () => {
+		expect(shallow.bind(shallow, <IdeaInfoOverlay />)).not.toThrow();
+	});
+
+	it('renders as expected', () => {
+		expect(shallow(<IdeaInfoOverlay />)).toMatchSnapshot();
+	});
+
+	describe('click close button', () => {
+		let overlay;
+
+		beforeEach(() => {
+			overlay = mount(<IdeaInfoOverlay/>);
+			overlay.find('.IdeaInfoOverlay-Close').simulate('click');
+			overlay.update();
```

Comment:
Interesting, why do you want the update? It doesn't do that automatically?

Reply:
u are right. This step is not necessary. Component is updated by itself in this case.